src/common/services/api/cart.service.js

```diff
@@ -134,6 +135,23 @@ function cart(cortexApiService){
       path: uri
     });
   }
+
+  function giftStartDate(day){
+    return cortexApiService.get({
+      path: ['nextdrawdate'],
+      cache: true
+    }).map((data) => {
+      let drawDate = data['next-draw-date'].split('-');
+      drawDate = new Date(drawDate[0], (drawDate[1] - 1), drawDate[2]);
```

Comment:
I believe if next-draw-date is 'YYYY-MM-DD', you can just pass it to new Date();